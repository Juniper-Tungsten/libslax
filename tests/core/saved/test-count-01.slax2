version 1.1;

template one match one () {
    expr "one";
}

template two match two ($three, $four) {
    expr "two";
}
