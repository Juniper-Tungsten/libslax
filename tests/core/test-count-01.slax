version 1.0;

template one match one {
    expr "one";
}

template two match two ($three, $four) {
    expr "two";
}
