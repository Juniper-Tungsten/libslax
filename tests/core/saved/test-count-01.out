<?xml version="1.0"?>

  one
  two
  3
  4
  5

